store-listing/st_ru.md

```diff
@@ -4,7 +4,23 @@
 OpenWebNet
 
 ### Short description
-TODO
+OpenWebNet это приложение для управления умным домом под системой Домотики My Home BTicino и Legrand
```

Comment:
@terranum could you please update the short description? Max length allowed is 80 characters.
Thanks!